pkg/scheduler/algorithm/priorities/test_util.go

```diff
@@ -58,7 +58,7 @@ func makeNodeWithExtendedResource(node string, milliCPU, memory int64, extendedR
 	}
 }
 
-func priorityFunction(mapFn PriorityMapFunction, reduceFn PriorityReduceFunction, metaData interface{}) PriorityFunction {
```

Comment:
Use priority map and reduce function here

Reply:
And change the tests which use this function as well.

Reply:
Change the function to return the actual result instead of another function, add the the required input arguments (```pod *v1.Pod, sharedLister schedulerlisters.SharedLister, nodes []*v1.Node```) and name the function something like ```runMapReducePriority```